src/app/space/analyze/analyze-overview/analyze-overview.component.ts

```diff
@@ -36,8 +45,12 @@ export class AnalyzeOverviewComponent implements OnInit, OnDestroy {
   }
 
   openForgeWizard(addSpace: TemplateRef<any>) {
-    this.selectedFlow = 'selectFlow';
-    this.modalRef = this.modalService.show(addSpace, {class: 'modal-lg'});
+    if (this.authService.isLoggedIn()) {
```

Comment:
There is no need to check on wizard launching now - that is covered by the github connect PR I have.